core-bundle/src/Image/ImageSizes.php

```diff
@@ -119,6 +139,19 @@ private function loadOptions(): void
 
         $options = [];
 
+        foreach ($this->predefinedSizes as $name => $imageSize) {
+            if (!isset($options['image_sizes'])) {
+                $options['image_sizes'] = [];
```

Comment:
I think this is no longer necessary as I had reverted the changes to `$GLOBALS['TL_CROP']`.